packages/localizations/src/sv-SE.ts

```diff
@@ -511,51 +514,56 @@ export const svSE: LocalizationResource = {
     },
     start: {
       actionLink: 'Logga in',
-      actionLink__use_email: undefined,
-      actionLink__use_phone: undefined,
+      actionLink__use_email: 'Använd e-post istället',
+      actionLink__use_phone: 'Använd telefon istället',
       actionText: 'Har du redan ett konto?',
       subtitle: 'för att fortsätta till {{applicationName}}',
       title: 'Skapa ditt konto',
     },
+    restrictedAccess: {
+      title: 'Begränsad åtkomst',
+      subtitle: 'Åtkomst till denna app är begränsad och en inbjudan krävs för att registrera sig.',
+      actionLink: 'Tillbaka till inloggning',
+    },
   },
   socialButtonsBlockButton: 'Fortsätt med {{provider|titleize}}',
-  socialButtonsBlockButtonManyInView: undefined,
+  socialButtonsBlockButtonManyInView: '{{provider|titleize}}',
   unstable__errors: {
-    already_a_member_in_organization: undefined,
+    already_a_member_in_organization: '{{email}} är redan medlem i organisationen.',
     captcha_invalid:
       'Registrering misslyckades på grund av säkerhetskontroller. Vänligen uppdatera sidan och försök igen eller kontakta supporten för mer hjälp.',
     captcha_unavailable:
       'Registrering misslyckades på grund av misslyckad bot-validering. Vänligen uppdatera sidan och försök igen eller kontakta supporten för mer hjälp.',
-    form_code_incorrect: 'Koden är felaktig.',
-    form_identifier_exists: 'Identifieraren finns redan.',
-    form_identifier_exists__email_address: 'E-postadressen finns redan.',
-    form_identifier_exists__phone_number: 'Telefonnumret finns redan.',
-    form_identifier_exists__username: 'Användarnamnet finns redan.',
-    form_identifier_not_found: 'Identifieraren hittades inte.',
-    form_param_format_invalid: 'Formatet är ogiltigt.',
+    form_code_incorrect: undefined,
```

Comment:
Thanks for your contribution! Is there any reason that some of these are now `undefined`? The current translations _seem_ to be correct.

Reply:
Ah, i was changing a lot to match the en-US one so it might have been that i made some of them undefined since they were that in the en-US.ts file. I was afraid I would break something if those were there. But if it dont those changes can be reverted and keep the translations that were correct.